src/Button/MeasureButton/MeasureButton.tsx

```diff
@@ -852,7 +858,13 @@ class MeasureButton extends React.Component<MeasureButtonProps> {
 
       if (geom instanceof OlGeomCircle) {
         measureTooltipCoord = geom.getLastCoordinate();
-        output = MeasureUtil.formatArea(geom, map, decimalPlacesInTooltips, geodesic);
+        if (!measureRadius) {
+          output = MeasureUtil.formatArea(geom, map, decimalPlacesInTooltips, geodesic);
+        } else {
+          const decimalHelper = Math.pow(10, decimalPlacesInTooltips);
+          const radius = Math.round(geom.getRadius() * decimalHelper) / decimalHelper;
+          output = `${radius.toString()} m`;
```

Comment:
Distances and areas are formatted by the use of `MeasureUtil.formatArea()` method that tries to get a [readable unit based on the actual area/distance](https://github.com/terrestris/ol-util/blob/main/src/MeasureUtil/MeasureUtil.ts#L165). To make the user experience the same behaviour we should make use of a similiar functionality here as well.

Reply:
Good point! I have just added that functionality